package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "license": "MIT",
   "engines": {
-    "node": "8.x",
-    "yarn": "1.x"
+    "node": ">=8",
+    "yarn": ">=1.0"
```

Comment:
I guess yarn would work also with just `>=1` (probably no need for the second semver character). Could you please change it?

Reply:
Will do!